CHANGELOG.md

```diff
@@ -12,8 +12,9 @@ We specialize in helping companies quickly and efficiently update client-side de
 Please follow the recommendations outlined at [keepachangelog.com](http://keepachangelog.com/). Please use the existing headings and styling as a guide, and add a link for the version diff at the bottom of the file. Also, please update the `Unreleased` link to compare to the latest release version.
 
 ## Versions
-### [Unreleased]
-Changes since the last non-beta release.
+### [13.4.1]
+#### Fixed
+- Fixed Typescript types for ServerRenderResult, ReactComponent, and RenderFunction interfaces. [PR 1582](https://github.com/shakacode/react_on_rails/pull/1582) by [kotarella1110](https://github.com/kotarella1110)
```

Comment:
I think we should keep those two lines for the unreleased version intact.

Reply:
Good call. Will fix.